spec/Section 4 -- Introspection.md

```diff
@@ -195,7 +198,7 @@ type __Directive {
   name: String!
   description: String
   locations: [__DirectiveLocation!]!
-  args: [__InputValue!]!
+  args(includeDeprecated: Boolean = false): [__InputValue!]!
```

Comment:
@IvanGoncharov @leebyron if we add this it would mean that we can now put the `deprecate` directive into a directive argument when using a directive. 

Is that something that was meant to be introduced here?

```graphql
directive @custom(if: bool @deprecate)
```

Reply:
In case directive arguments cannot be deprecated, we will also need a new type for them, as `__InputValue` contains `isDeprecated` and `deprecationReason`

Reply:
Apparently we can already do this #907